coalib/parsing/Filter.py

```diff
@@ -0,0 +1,22 @@
+from coalib.parsing.filters.LanguageFilter import language_filter
+
+
+class Filter:
+    all_filters = {'language': language_filter}
```

Comment:
I think to modularize things more, you should provide a function `registerFilter` where someone can register his own filter object/class/function/etc. Then we can dynamically built a filter with different sets of functionality (though I cannot think of a useful case right now).

This would work for example like https://docs.python.org/3/library/unittest.html#unittest.TestCase.addTypeEqualityFunc